src/pages/index.tsx

```diff
@@ -8,12 +8,14 @@ const Home: React.FC = () => {
   if (status === 'authenticated') {
     // Then redirect the user to the app pages
```

Comment:
add next router push

Reply:
Why would we need that?
FYI, I didn't comment under the if-statement. idk who did it, but not me.